### DISCUSSION TITLE
save output automatically to hard drive

### DISCUSSION THREAD

**Discussion by kozig** [lllyasviel/Fooocus#1160] [Q&A] [answered]
I tried making a python script using selenium but is there another way to just output the generated images to a local drive? Any advice or direction is greatly appreciated, I don't know what angle to approach this from.

**Comment by robomaniac** [comment]
All the images rendered are on your hard drive. Right click an image and click open in new tab. Check the address in address bar. You will have the C:/ directory where it is on computer. Then you go back in the repertoire and you can search for *.png and it will show every image you created with focuuus

**Comment by alfred-delacosta** [comment, marked as answer]
All the images are saved locally to wherever you are running the app in the following folder \Fooocus\outputs. For example, I am running mine in my user profile (I'm on Windows) so the location of my pictures would be: C:\Users\Me\Fooocus_win64_2-1-791\Fooocus\outputs

If you are looking to automatically transfer the images to another hard drive, external drive, or flash drive, you will need to write a program or a powershell script that will "watch" that output folder and copy anything that gets placed there into the location where you want it to save.

**Reply by kozig** [reply]
Thanks I was manually saving the images from the webUI. This simplifies things.
